naslib/defaults/trainer.py

```diff
@@ -57,8 +61,8 @@ def __init__(self, optimizer, config, lightweight_output=False):
         self.val_loss = utils.AverageMeter()
 
         n_parameters = optimizer.get_model_size()
-        # logger.info("param size = %fMB", n_parameters)
-        self.search_trajectory = utils.AttrDict(
+        logger.info("param size = %fMB", n_parameters)
+        self.errors_dict = utils.AttrDict(
```

Comment:
I would keep the name as "search_trajectory", since it also contains non-error metrics such as accuracies, train_time and params